akka-actor-typed-tests/src/test/java/jdocs/akka/typed/GracefulStopDocTest.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package jdocs.akka.typed;
+
+//#imports
+import akka.actor.typed.ActorSystem;
+import akka.actor.typed.Behavior;
+import akka.actor.typed.PostStop;
+import akka.actor.typed.javadsl.Behaviors;
+import scala.concurrent.Await;
+import scala.concurrent.duration.Duration;
+import java.util.concurrent.TimeUnit;
+
+//#imports
+
+public class GracefulStopDocTest {
+
+    //#master-actor
+
+    public abstract static class JobControl {
+        // no instances of this class, it's only a name space for messages
+        // and static methods
+        private JobControl() {
+            // no instances of this class, it's only a name space for messages
+            // and static methods
```

Comment:
Good, but maybe one comment saying that is enough? ;)

Reply:
https://github.com/akka/akka/commit/f3d11cbc4b91eb0dd3697f88f58c62de83113808 shall I drop the lower one?